src/OrdersList/hooks/useBuildQuery/useBuildQuery.test.js

```diff
@@ -3,15 +3,32 @@ import queryString from 'query-string';
 import { renderHook } from '@folio/jest-config-stripes/testing-library/react';
 import { ORDER_STATUSES } from '@folio/stripes-acq-components';
 
+import { CUSTOM_FIELDS } from '../../../../test/jest/fixtures/customFields';
```

Comment:
just `fixtures/customFields`

Reply:
Thanks for the hint, using now just `import { CUSTOM_FIELDS } from 'fixtures/customFields';`